docs/release-notes/self-hosted/self-hosted-v12.0.0.md

```diff
@@ -0,0 +1,128 @@
+---
+rss_title: Codacy release notes RSS feed
+rss_href: /feed_rss_created.xml
+description: Release notes for Codacy Self-hosted v12.0.0.
+codacy_tools_version_old: https://github.com/codacy/codacy-tools-sh/releases/tag/sh-1.1.6
+codacy_tools_version_new: https://github.com/codacy/codacy-tools-sh/releases/tag/sh-1.2.2
+---
+
+# Self-hosted v12.0.0
+
+These release notes are for [Codacy Self-hosted v12.0.0](https://github.com/codacy/chart/releases/tag/12.0.0){: target="_blank"}, released on June 21, 2023.<!-- TODO Update release date -->
+
+📢 [Visit the Codacy roadmap](https://roadmap.codacy.com) and <span class="skip-vale">let us know</span> your feedback on both new and planned product updates!
+
+## Upgrading Codacy Self-hosted
+
+Follow the steps below to upgrade to Codacy Self-hosted v12.0.0:
+
+1.  Check the [release notes for all Codacy Self-hosted versions](../index.md#self-hosted) **between your current version and the most recent version** for breaking changes and follow the instructions provided <span class="skip-vale">carefully</span>.
+
+1.  Follow the instructions to [upgrade your Codacy Self-hosted instance](https://docs.codacy.com/v12.0/chart/maintenance/upgrade/).
+
+1.  Update your Codacy command-line tools to the following versions:
+
+    -   [Codacy Analysis CLI 7.8.3](https://github.com/codacy/codacy-analysis-cli/releases/tag/7.8.3)
+    -   [Codacy Coverage Reporter 13.13.1](https://github.com/codacy/codacy-coverage-reporter/releases/tag/13.13.1)
+
+## Breaking changes
+
+-   If you have set `scheduler.enable: "true"` in `values.yaml`, do the following before updating Kubernetes.
+
+    Assuming you're using the default `codacy` namespace:
+
+    1.  Remove all `scheduler.*` settings from `values.yaml`
+
+    1.  Update the ConfigMap of `worker-manager`:
+
+        ```bash
+        kubectl patch configmap/worker-manager --namespace codacy --type merge --patch '{"data":{"CONFIG_FORCE_codacy_kubernetes_scheduler_enable": "false","CONFIG_FORCE_codacy_kubernetes_scheduler_name":"default-scheduler"}}'
+        ```
+
+    1.  Restart the `worker-manager` Deployment:
+
+        ```bash
+        kubectl rollout restart deployment/worker-manager -n codacy
+        ```
+
```

Comment:
Correct resource name.
@tfrgoncalves Please validate. 🙏 

```suggestion
    1.  Update the ConfigMap of `codacy-worker-manager`:

        ```bash
        kubectl patch configmap/codacy-worker-manager --namespace codacy --type merge --patch '{"data":{"CONFIG_FORCE_codacy_kubernetes_scheduler_enable": "false","CONFIG_FORCE_codacy_kubernetes_scheduler_name":"default-scheduler"}}'
        ```

    1.  Restart the `codacy-worker-manager` Deployment:

        ```bash
        kubectl rollout restart deployment/codacy-worker-manager -n codacy
        ```

```

Reply:
@tfrgoncalves Please validate. 🙏

Reply:
LGTM. Thank you @Ngwerume for the find and @claudiacarpinteiro for the fix.